docs/usage.rst

```diff
@@ -105,9 +106,21 @@ To validate a RAML file with Python:
    >>> validate(RAML_FILE)
    InvalidRootNodeError: RAML File does not define an API title.
 
+.. code-block:: python
+
+   >>> from ramlfications import validate
+   >>> RAML_FILE = "/path/to/invalid/no-base-uri-no-title.raml"
+   >>> validate(RAML_FILE)
+   MultipleValidationErrors:
```

Comment:
(continuing thought) - you'd have to probably rename the current `InvalidRAMLError` to something like `BaseRAMLError`.